setup.py

```diff
@@ -1,5 +1,24 @@
 #!/usr/bin/env python3
 
-from robotpy_build.setup import setup
 
-setup()
+from robotpy_build.setup import Setup
+
+s = Setup()
+s.prepare()
+
+try:
+    from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+except ImportError:
+    pass
+else:
+
+    class bdist_wheel(_bdist_wheel):
```

Comment:
This should be added to robotpy-build instead?

Reply:
Probably, yeah. This was more a PoC than anything I expected to actually be merged. Would be good to get rid of `setup.py` everywhere anyway.

I'm not too sure how to best integrate this into robotpy-build proper though.